libs/core-components/src/lib/Button/Button.tsx

```diff
@@ -24,27 +21,33 @@ export const SIZES = [''].concat(Object.keys(SIZE_MAP));
 
 export const ATTRIBUTES = ['button', 'submit', 'reset'];
 
-function isNotEmptyString(props, propName, componentName) {
-  if (!props[propName] || props[propName].replace(/ /g, '') === '') {
-    return new Error(`No text passed to ${componentName}. Validation failed.`);
-  }
-  return null;
-}
```

Comment:
I like dropping this.

<details>

If user doesn't pass in button content (this function was used by `children` propType), then just show no content. Missing button content is a user error that should be caught during testing. I thought it was a bit overkill when I noticed it.

</details>